modules/auxiliary/dos/misc/ibm_tsm_dos.rb

```diff
@@ -72,7 +72,7 @@ def run
     print_status("Packet sent!")
   rescue Rex::AddressInUse, ::Errno::ETIMEDOUT, Rex::HostUnreachable, Rex::ConnectionTimeout, Rex::ConnectionRefused, ::Timeout::Error, ::EOFError => ex
     print_error("Exploit failed: #{ex.class} #{ex.message}")
-    elog("#{ex.class} #{ex.message}\n#{ex.backtrace * "\n"}")
+    elog(error: e)
```

Comment:
I believe this will break now as the first arg isn't supplied, I think these types of calls will need to be replaced with:
```
elog(e)
```